src/components/TitleTransferPanel/TitleView/Holder.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import css from "./TitleView.module.scss";
+import { Address } from "../Address";
+import { AddressLink } from "./../AddressLink";
+import { ChangeHolder } from "../Actions/ChangeHolder";
```

Comment:
Might want to merge the code from ChangeHolder to this file since there's not much gain from putting it separate. Doesnt help that ChangeHolder component isnt going to be reused elsewhere without this wrapper.

Reply:
Same as the other "TitleView". Will suggest to rename `TitleViews` to to `ActionPanels` . And their components to be `HolderActionPanel`, `BeneficiaryActionPanel`